Utilities/MCStepLogger/CMakeLists.txt

```diff
@@ -33,4 +61,21 @@ if (HAVESIMULATION)
   endif()
     set_tests_properties(mcloggertest PROPERTIES ENVIRONMENT ${PRELOAD}=${CMAKE_BINARY_DIR}/lib/libMCStepLogger${CMAKE_SHARED_LIBRARY_SUFFIX})
     set_property(TEST mcloggertest APPEND PROPERTY ENVIRONMENT VMCWORKDIR=${CMAKE_SOURCE_DIR})
+
+  # check whether StepLogger output can be written to ROOT file
+  # fix output file name for logged data
+  set(STEPLOGGER_ROOTFILE "MCStepLoggerOutput_test.root")
+  add_test(NAME mcloggertest_tofile COMMAND ${CMAKE_BINARY_DIR}/bin/runTPC -n 1 -e TGeant3)
+  set_tests_properties(mcloggertest_tofile PROPERTIES ENVIRONMENT ${PRELOAD}=${CMAKE_BINARY_DIR}/lib/libMCStepLogger${CMAKE_SHARED_LIBRARY_SUFFIX})
+  # set environment accordingly
+  set_property(TEST mcloggertest_tofile APPEND PROPERTY ENVIRONMENT VMCWORKDIR=${CMAKE_SOURCE_DIR} MCSTEPLOG_TTREE=1 MCSTEPLOG_OUTFILE=${STEPLOGGER_ROOTFILE})
+
+  # check for working analysis
+  # fix output file name for analysis
+  set(MCANALYSIS_ROOTFILE "BasicMCAnalysis.root")
+  add_test(NAME basicmcanalysis COMMAND ${CMAKE_BINARY_DIR}/bin/mcStepAnalysis analyze -f ${STEPLOGGER_ROOTFILE} -o ${MCANALYSIS_ROOTFILE} -l testLabel)
+  set_tests_properties(basicmcanalysis PROPERTIES DEPENDS mcloggertest_tofile)
+  # set environment accordingly
+  set_property(TEST basicmcanalysis APPEND PROPERTY ENVIRONMENT VMCWORKDIR=${CMAKE_SOURCE_DIR})
```

Comment:
There are now also 2 tests added
1) checking whether StepLogger can flush to a ROOT file
2) running the BasicMCAnalysis on the ROOT file produced in 1) (==> only executed if 1) passes)